Automattic/newspack-sponsors
# feat: update sponsors icon to Material Design

### All Submissions:






### Changes proposed in this Pull Request:

Instead of using Dashicons, I've switched the Plugin icon Material's `monetization_on`.
Also fixed the casing for Sponsor Logo to match Featured Image (minor change).

![Screenshot 2020-08-21 at 17 13 37](https://user-images.githubusercontent.com/177929/90911963-aee2f180-e3d1-11ea-90d0-d9b7b7308bcb.png)

### How to test the changes in this Pull Request:

1. Check the wpadmin sidebar icon
2. Same whilst adding a new sponsor
3. Switch to this branch
4. In wpadmin you should see the new icon and in the editor you houldn't see an icon after "Sponsor Settings" 

### Other information:
